# BGI-Alzheimer-s-Hippocampal-Atlas
The source code for passage "Molecular Pathways and Diagnosis in Spatially Resolved Alzheimer’s Hippocampal Atlas"
![image](https://github.com/qyTao185/BGI-Alzheimer-s-Hippocampal-Atlas/blob/main/workflow.png)
## Summary
We employed advanced high-resolution Stereo-seq, combined with single-nucleus sequencing techniques, to investigate gene expression and cell composition changes in the hippocampus in individuals with or without Alzheimer's disease (AD). The resulting spatial transcriptomic map, achieved with single-cell precision, unveiled AD-associated alterations with spatial specificity, including: 1) heightened expression of synapse pruning genes in the Fimbria of AD, accompanied by a significant increase in disease activated microglia (DAM) and disease activated astrocyte (DAA) proportion, 2) elevated expression of energy generation genes in CA4, where mitochondrial complexes-encoded genes elevated their expression in EX_CA4 and 3) Aβ plaques aggregated in CA1 and SLRM subregions, integration of the transcriptomic map with Aβ immunohistochemical staining revealed a sequential enrichment of microglia and astrocytes around Aβ plaques. Furthermore, we observed significant reduction in brain-derived extracellular vesicles carrying CCK and PMP2 in AD plasma, highlighting their potential as robust diagnostic tools for clinical applications.

## Data availability
Stereo-seq data：16chips(AD: 4 smaples; Control: 4 smaples;  Two consecutive slices per sample )
Single-nucleus RNA sequencing data 
All raw data have been deposited to CNGB Nucleotide Sequence Archive (accession code: CNP0005077, https://db.cngb.org/search/project/CNP0005077/) and are publicly available as of the date of publication. Accession numbers are listed in the key resources table. 
